### PR TITLE
Make the "Ballerina Tool" Wording Consistent in Docs

### LINKED ISSUE #1563: Use consistent wordings when referring the ballerina tool
**Description:**
The following words are used to ballerina tool in different places in the docs (ex. in pages like [1],[2])
`Ballerina Tool` 
 `Ballerina tool` 
 `ballerina` Tool

Use a common way to refer the tool accross all the pages.

[1] https://ballerina.io/swan-lake/learn/using-the-cli-tools/
[2] https://ballerina.io/swan-lake/learn/keeping-ballerina-up-to-date/

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Make the "Ballerina Tool" wording consistent in Docs.

> Fixes #1563

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: 1.0/learn/how-to-generate-code-for-protocol-buffers/index.html [hunk 1]
```diff
@@ -211,7 +211,7 @@
                               <div class="section">
                                  <h1 id="how-to-generate-ballerina-code-for-protocol-buffer-definition">How to generate Ballerina code for Protocol Buffer Definition</h1>
 
-<p>The <code class="highlighter-rouge">Protocol Buffers</code> to Ballerina tool provides capabilities to generate Ballerina source code for the Protocol
+<p>The <code class="highlighter-rouge">Protocol Buffers</code> to Ballerina Tool provides capabilities to generate Ballerina source code for the Protocol
```
Comment (MaryamZi) [inline 1.0/learn/how-to-generate-code-for-protocol-buffers/index.html:214]: In these contexts it's just another tool, right? I wonder if we should leave these as is and use "Ballerina Tool" only for the CLI tool that allows the `ballerina` commands?

File: 1.1/learn/how-to-keep-ballerina-up-to-date/index.html [hunk 2]
```diff
@@ -255,7 +255,7 @@ <h2 id="terminology">Terminology</h2>
```
Comment (MaryamZi) [inline 1.1/learn/how-to-keep-ballerina-up-to-date/index.html:255]: Shouldn't we update L249?

"Update the Ballerina Tool"

File: 1.1/learn/how-to-keep-ballerina-up-to-date/index.html [hunk 3]
```diff
@@ -334,13 +334,13 @@ <h3 id="the-active-distribution">The “active” distribution</h3>
 
 <ul>
   <li>One only distribution from the above list can be active at a given time.</li>
-  <li>Ballerina tool delegates most of the user requests to the active distribution. The commands such as build, test, run, pull, and push are delegated to the active distribution, while the commands such as dist and version are handled by the tool itself.  E.g., when you invoke <code class="highlighter-rouge">ballerina build</code>, the Ballerina tool dispatches this request to the active distribution.</li>
+  <li>Ballerina tool delegates most of the user requests to the active distribution. The commands such as build, test, run, pull, and push are delegated to the active distribution, while the commands such as dist and version are handled by the tool itself.  E.g., when you invoke <code class="highlighter-rouge">ballerina build</code>, the Ballerina Tool dispatches this request to the active distribution.</li>
```
Comment (MaryamZi) [inline 1.1/learn/how-to-keep-ballerina-up-to-date/index.html:337]:
```suggestion
  <li>Ballerina Tool delegates most of the user requests to the active distribution. The commands such as build, test, run, pull, and push are delegated to the active distribution, while the commands such as dist and version are handled by the tool itself.  E.g., when you invoke <code class="highlighter-rouge">ballerina build</code>, the Ballerina Tool dispatches this request to the active distribution.</li>
```

File: 1.1/learn/how-to-keep-ballerina-up-to-date/index.html [hunk 4]
```diff
@@ -452,10 +452,10 @@ <h3 id="pull-a-specific-distribution">Pull a specific distribution</h3>
 Downloading jballerina-1.0.3 100% <span class="o">[==================================]</span> 96/96 MB <span class="s1">'jballerina-1.0.3'</span> successfully <span class="nb">set </span>as the active distribution
 </code></pre></div></div>
 
-<h3 id="update-the-ballerina-tool">Update the Ballerina tool</h3>
+<h3 id="update-the-ballerina-tool">Update the Ballerina Tool</h3>
 
 <ul>
-  <li>The <code class="highlighter-rouge">ballerina update</code> command updates the Ballerina tool itself to the latest version. Ballerina tool versions are independent from distribution versions. We expect these tool updates to be rare compared to distribution releases.</li>
+  <li>The <code class="highlighter-rouge">ballerina update</code> command updates the Ballerina Tool itself to the latest version. Ballerina tool versions are independent from distribution versions. We expect these tool updates to be rare compared to distribution releases.</li>
```
Comment (MaryamZi) [inline 1.1/learn/how-to-keep-ballerina-up-to-date/index.html:458]:
```suggestion
  <li>The <code class="highlighter-rouge">ballerina update</code> command updates the Ballerina Tool itself to the latest version. Ballerina Tool versions are independent from distribution versions. We expect these tool updates to be rare compared to distribution releases.</li>
```

File: learn/keeping-ballerina-up-to-date.md [hunk 5]
```diff
@@ -230,5 +230,5 @@ Downloading ballerina-tool-0.8.1 100% [====================================] 1/1
 
 Updated to latest tool version: 0.8.1
 Cleaning old files...
-Ballerina tool updated successfully
+Ballerina Tool updated successfully
```
Comment (MaryamZi) [inline learn/keeping-ballerina-up-to-date.md:233]: This is probably the output from the `ballerina update` command. So we probably have to fix it there (first?). @keizer619

File: swan-lake/learn/using-the-cli-tools.md [hunk 6]
```diff
@@ -12,13 +12,13 @@ redirect_from:
   - /swan-lake/learn/using-the-cli-tools
 ---
 
-## Using the 'ballerina' Tool
+## Using the Ballerina Tool
 
-**Ballerina** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
+**`ballerina`** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
```
Comment (MaryamZi) [inline swan-lake/learn/using-the-cli-tools.md:17]: This change isn't consistent with https://github.com/ballerina-platform/ballerina-dev-website/pull/1585/files#diff-13dab7a3e9ee8fe33dd736b9ed8c9ff722c937b0c7777610128b048dc83ccd38R260.

File: 1.0/learn/how-to-generate-code-for-protocol-buffers/index.html (hunk 1 above)
Comment (praneesha) [reply 1.0/learn/how-to-generate-code-for-protocol-buffers/index.html]: Fixed.

File: 1.1/learn/how-to-keep-ballerina-up-to-date/index.html (hunk 2 above)
Comment (praneesha) [reply 1.1/learn/how-to-keep-ballerina-up-to-date/index.html:255]: Done.

File: swan-lake/learn/using-the-cli-tools.md (hunk 6 above)
Comment (praneesha) [reply swan-lake/learn/using-the-cli-tools.md]: Fixed.

File: 1.1/learn/how-to-keep-ballerina-up-to-date/index.html (hunk 3 above)
Comment (praneesha) [reply 1.1/learn/how-to-keep-ballerina-up-to-date/index.html]: Fixed.

File: 1.1/learn/how-to-keep-ballerina-up-to-date/index.html (hunk 4 above)
Comment (praneesha) [reply 1.1/learn/how-to-keep-ballerina-up-to-date/index.html]: Fixed.

File: learn/keeping-ballerina-up-to-date.md (hunk 5 above)
Comment (praneesha) [reply learn/keeping-ballerina-up-to-date.md:233]: Created https://github.com/ballerina-platform/ballerina-lang/issues/27407 to fix in the Tool outputs.

File: learn/keeping-ballerina-up-to-date.md [hunk 7]
```diff
@@ -87,12 +87,12 @@ Now that you are familiar with the terminology, let’s look at how you can keep
 ### The "active" Distribution
 
 - One only distribution from the above list can be active at a given time.
-- Ballerina tool delegates most of the user requests to the active distribution. The commands such as build, test, run, pull, and push are delegated to the active distribution, while the commands such as dist and version are handled by the tool itself.  E.g., when you invoke `ballerina build`, the Ballerina tool dispatches this request to the active distribution.
+- Ballerina Tool delegates most of the user requests to the active distribution. The commands such as build, test, run, pull, and push are delegated to the active distribution, while the commands such as dist and version are handled by the tool itself.  E.g., when you invoke `ballerina build`, the Ballerina Tool dispatches this request to the active distribution.
```
Comment (MaryamZi) [inline learn/keeping-ballerina-up-to-date.md:90]: Shouldn't this be 
```suggestion
- The Ballerina Tool delegates most of the user requests to the active distribution. The commands such as build, test, run, pull, and push are delegated to the active distribution, while the commands such as dist and version are handled by the tool itself.  E.g., when you invoke `ballerina build`, the Ballerina Tool dispatches this request to the active distribution.
```
for consistency?

File: swan-lake/learn/keeping-ballerina-up-to-date.md [hunk 8]
```diff
@@ -71,15 +71,15 @@ Once the installation is complete, you would see the directory structure below i
...
   
-  E.g., when you invoke `ballerina build`, the Ballerina tool dispatches this request to the active distribution.
+  E.g., when you invoke `ballerina build`, the Ballerina Tool dispatches this request to the active distribution.
 
 You can [change this active distribution](#changing-the-active-distribution) at any time or manage it using the Ballerina Tool. However, first, you need to update the Ballerina Tool to its latest version.
 
 ### Updating the Ballerina Tool
 
-The `ballerina update` command updates the Ballerina Tool itself to the latest version. Ballerina Tool versions are independent from the Ballerina distribution versions. These tool updates are expected to be rare compared to distribution releases.
+The `ballerina update` command updates the Ballerina Ttool itself to the latest version. Ballerina Tool versions are independent from the Ballerina distribution versions. These tool updates are expected to be rare compared to distribution releases.
```
Comment (MaryamZi) [inline swan-lake/learn/keeping-ballerina-up-to-date.md:82]:
```suggestion
The `ballerina update` command updates the Ballerina Tool itself to the latest version. Ballerina Tool versions are independent from the Ballerina distribution versions. These tool updates are expected to be rare compared to distribution releases.
```

File: swan-lake/learn/using-the-cli-tools.md [hunk 9]
```diff
@@ -5,20 +5,20 @@ description: Learn all the command line interface (CLI) commands need to get sta
...
   - /swan-lake/learn/cli-commands
   - /swan-lake/learn/cli-commands/
   - /swan-lake/learn/using-the-cli-tools
 ---
 
-## Using the 'ballerina' Tool
+## Using the Ballerina Tool
 
-**Ballerina** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
+**`Ballerina`** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
```
Comment (anupama-pathirage) [inline swan-lake/learn/using-the-cli-tools.md:17]: Is it correct to use `Ballerina` here? Note the upper case `B`. Instead should we reword it to "The `ballerina` is a .."

Comment (MaryamZi) [reply swan-lake/learn/using-the-cli-tools.md:17]: Shouldn't we say 

```suggestion
The Ballerina Tool is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
```

for consistency?

**Review by MaryamZi — APPROVED** [review]

**Review by anupama-pathirage — APPROVED** [review]
